templates/assignee_no_login_nag.html

```diff
@@ -0,0 +1,29 @@
+<p>
+    The following {{ plural('bug assigned to the following person has', data, pword='bugs assigned to the following people have') }} been unassigned due to inactivity:
+</p>
+<table {{ table_attrs }}>
+    <thead>
+        <tr>
+            <th>Bug</th>
+            <th>Summary</th>
+            <th>Component</th>
+            <th>Assignee</th>
+            <th>Assignee Status</th>
+            <th>Triage Owner</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for bug in data %}
+            <tr>
+                <td>
+                    <a href="https://bugzilla.mozilla.org/show_bug.cgi?id={{ bug.id }}">{{ bug.id }}</a>
+                </td>
+                <td>{{ bug.summary | e }}</td>
```

Comment:
I do not think `bug.summary` would work. Did you test that?

Reply:
I used the template `assignee_no_login.html` as reference, see: https://github.com/mozilla/bugbot/blob/67eaf1db7d8b1186b1eca70244c52f6e1990b162/templates/assignee_no_login.html#L13-L28, as it uses `summary`. I'll make changes to `assignee_no_login_nag.html` to follow `assignee_no_login.html` more closely.

Reply:
Done in f322dc789ad462fa2b7941bbf28d7dccbec30322!